src/level-ext.adoc

```diff
@@ -91,13 +91,15 @@ endif::[]
 11+| _Reserved bits for future extensions must be 1 so they are implicitly granted_
 |Bits[4:3]| R | W | C | LM | EL | SL    | X | ASR | Mode^1^ |
 | 0-2   10+| reserved
-| 3       | ✔ |   | ✔ | ✔  | ✔  | N/A   |   |     | N/A | Data & Cap R0
+| 3       | ✔ |   | ✔ | ✔  | ✔  | ∞^1^  |   |     | N/A | Data & Cap R0
```

Comment:
```suggestion
| 3       | ✔ |   | ✔ | ✔  | ✔  | 0^1^  |   |     | N/A | Data & Cap R0
```

Reply:
IMO this should be zero for consistency: just report the negation of bit 0 (and bit 0-1 for multiple level bits).

Reply:
This one can load a W cap, and so its SL would be important if we add a transitive SL. In an earlier sketch we had all combinations of SL and W=0 in this quadrant for this reason.

I think we decided it was fine to reserve 3/4 of these now (and just have the one, 3 or 0). Whichever one we have selected here with the bit pattern is what we should report. I think that is (as Alex suggests) 0, to match the W=1 case. We can add 1/2/3 if required. This is slightly unlike Quad 1, where I expect to have infinity regardless of how many levels there are.

Reply:
Yes I agree if we end up adding the transitive property, then 2 should report SL=1, and for n bit we add 1 with SL=2 and zero with SL=3.